src/MachO/BinaryParser.tcc

```diff
@@ -199,7 +199,6 @@ void BinaryParser::parse_load_commands(void) {
       // DyLib Command
       // =============
       case LOAD_COMMAND_TYPES::LC_LOAD_WEAK_DYLIB:
-      case LOAD_COMMAND_TYPES::LC_ID_DYLIB:
```

Comment:
Why removing this entry from the parser ?

Reply:
- Similar to the above, there is no need to add ID_DYLIB command to binary_->libraries_. Just similar as the LLVM src, there is also no more processing for LC_ID_DYLIB.
- However, a new container may still be needed to store all LoadCommands for print purpose, but preferably not binary_->libraries_.

Reply:
I agree with both points but here you break the support of ``LC_ID_DYLIB``

Reply:
You are right. It is really not good to simply delete a case.

By comparing Binary.cpp and BinaryParser.hpp, I found that Binary.cpp will enable `binary_->commands_` to store all load commands (including LC_ID_DYLIB), but BinaryParser.hpp does not enable the container. 

Maybe it would be better to enable `commands_` to store(support) LC_ID_DYLIB in BinaryParser.hpp? 

I'm not sure.